packages/rancher-prom-to-teams/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,8 @@
+--- charts-original/Chart.yaml
```

Comment:
Can we rename this package to rancher-prom2teams? To be in-line with the upstream chart